### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -84,50 +92,41 @@ ynh_script_progression --message="Configuring NGINX web server..." --weight=3
...
 # SPECIFIC SETUP
 #==============================================
 # INSTALL DROPPY
 #==============================================
 ynh_script_progression --message="Building $app... (this will take some time and resources!)" --weight=26
 
-pushd "$final_path" || ynh_die
+pushd "$final_path"
 	ynh_use_nodejs
 	ynh_exec_warn_less ynh_npm install
```
Comment (tituspijean) [inline scripts/install]:
```suggestion
	ynh_exec_warn_less $ynh_npm install
```
`ynh_npm` is actually a variable to npm's binary. ;)

File: scripts/upgrade [hunk 2]
```diff
@@ -107,62 +106,50 @@ ynh_exec_warn_less ynh_install_nodejs --nodejs_version=$nodejs_version
...
 #==============================================
+
 if [ "$upgrade_type" == "UPGRADE_APP" ]
 then
 	ynh_script_progression --message="Building $app... (this will take some time and resources!)" --weight=25
 
-	pushd "$final_path" || ynh_die
+	pushd "$final_path"
 		ynh_use_nodejs
 		ynh_exec_warn_less ynh_npm install
```
Comment (tituspijean) [inline scripts/upgrade]:
```suggestion
		ynh_exec_warn_less $ynh_npm install
```

**Review by tituspijean — CHANGES_REQUESTED** [review]

File: scripts/install (hunk 1 above)
Comment (tituspijean) [reply scripts/install]: It looks it works well in YunoHost 4.2.x, but the `$` is required for v4.3...?

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**